flight/Libraries/sanitycheck.c

```diff
@@ -211,30 +207,35 @@ DONT_BUILD_IF(MANUALCONTROLSETTINGS_STABILIZATION1SETTINGS_NUMELEM != MANUALCONT
  * @param[in] index Which stabilization mode to check
  * @returns SYSTEMALARMS_CONFIGERROR_NONE or SYSTEMALARMS_CONFIGERROR_MULTIROTOR
  */
-static int32_t check_stabilization_settings(int index, bool multirotor,
-		SharedDefsThrustModeStabBankOptions thrust_mode)
+static int32_t check_stabilization_settings(int index, bool multirotor)
 {
 	uint8_t modes[MANUALCONTROLSETTINGS_STABILIZATION1SETTINGS_NUMELEM];
+	uint8_t thrust_mode;
```

Comment:
This change is unrelated:   thrust_mode is factored in here to simplify enclosing call/switch statement.